optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/DefaultPartitionedSearchPhase.java

```diff
@@ -48,10 +50,10 @@
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
 public class DefaultPartitionedSearchPhase<Solution_> extends AbstractPhase<Solution_>
-        implements PartitionedSearchPhase<Solution_>, PartitionedSearchPhaseLifecycleListener<Solution_> {
+        implements PartitionedSearchPhase<Solution_> {
```

Comment:
Let's keep PartitionedSearchPhaseLifecycleListener for now, due to consistency across algorithms LocalSearchPhase, ConstructionHeuristicsPhase, ...